react/features/mobile/navigation/rootNavigationContainerRef.js

```diff
@@ -13,7 +17,32 @@ export const rootNavigationRef = React.createRef();
  * @returns {Function}
  */
 export function navigateRoot(name: string, params: Object) {
-    // $FlowExpectedError
     return rootNavigationRef.current?.navigate(name, params);
 }
 
+/**
+ * User defined navigation action included inside the reference to the container.
+ *
+ * @returns {Function}
+ */
+export function goBack() {
+    return rootNavigationRef.current?.goBack();
+}
+
+/**
+ * Navigates back to Welcome page, if it's available.
+ *
+ * @param {Object|Function} stateful - Either the whole Redux state object or the Redux store's {@code getState} method.
+ * @param {Function} dispatch - Redux dispatch function.
+ * @returns {void}
+ */
+export function goBackToRoot(stateful: Function | Object, dispatch: Function) {
```

Comment:
Looks like we really only need this function in a single place (appNavigate) since we should use that function whenever we need to "hang up" in order to avoid leaving dangling tracks or connections. Let's inline it, since it's only used / needed there.